src/communication/xcSession.js

```diff
@@ -45,17 +61,38 @@ define(["../javascriptHelper", "../guid", "./xcWebSocketPublisher", "./xcWebSock
 
         Session.prototype.createPublisher = function () {
             var publisher = new Publisher(this.webSocket, this.configuration, this.privateTopic);
+            this.publishers.push(publisher);
             return publisher;
         }
 
 
         Session.prototype.createSubscriber = function () {
             var subscriber = new Subscriber(this.webSocket, this.configuration, this.replyPublisher, this.guid);
+            this.subscribers.push(subscriber);
             return subscriber;
         }
 
+
+        Session.prototype.clearPublishers = function () {
+            this.publishers = [this.replyPublisher];
+        }
+
+
+        Session.prototype.clearSubscribers = function () {
+            this.subscribers = [];
+        }
+
+
+        Session.prototype.dispose = function () {
```

Comment:
Not really useful
better create functions such as disposePublisher(publisher) and disposeSubscriber(subscriber)